src/Sinch/Core/Http.cs

```diff
@@ -128,7 +128,7 @@ internal class Http : IHttp
                     // will not retry when no "expired" header for a token.
                     const string wwwAuthenticateHeader = "www-authenticate";
                     if (_auth.Scheme == AuthSchemes.Bearer && result.Headers.Contains(wwwAuthenticateHeader) &&
-                        !result.Headers.GetValues(wwwAuthenticateHeader).Contains("expired"))
+                        !result.Headers.GetValues(wwwAuthenticateHeader).Any(x => x.Contains("expired")))
```

Comment:
Looking at the fix, it highlights that our expiration detection mechanism is not robust enough: any change on the backend side for the property `error_description` will break our SDKs.
For instance, if the message changes for `Jwt expiry reached at ...`, this won't work anymore

Reply:
Can we merge that while we settle on the robust way of doing that?

Reply:
This commit fixes the reported bug indeed